docs/core/versions/index.md

```diff
@@ -13,89 +13,220 @@ ms.assetid: f6f684b1-1d2c-4105-8376-7c1959e23803
 
 # .NET Core Versioning
 
-.NET Core is a platform of [NuGet packages](../packages.md), of frameworks and distributed as a unit. Each of these platform layers can be versioned separately for product agility and to accurately describe product changes. While there is significant versioning flexibility, there is a desire to version the platform as a unit to make the product easier to understand.
+.NET Core is made of [NuGet packages](../packages.md), tools, and frameworks that are distributed as a unit. Each of these platform layers can be versioned separately, enabling better agility. While there is significant versioning flexibility in that regard, we also have a desire to version the platform as a unit to make the product easier to understand.
 
-The product is in some respects unique, being described and delivered via a package manager (NuGet) as packages. While you typically acquire .NET Core as a standalone SDK, the SDK is largely a convenience experience over NuGet packages and therefore not distinct from packages. As a result, versioning is first and foremost in terms of packages and other versioning experiences follow from there.
+This article aims at clarifying how the .NET Core SDK and runtime are versioned.
 
-## Semantic Versioning
+## Summary
+
+There are lots of moving parts that version independently in .NET Core, but starting with .NET core 2.0, we'll make sure that there is an easy to understand top-level version number that everybody understands to be *the* version of ".NET Core" as a whole. The rest of this document goes into the details of the versioning of all those parts, but you shouldn't need to know all those in order to use .NET Core, or build libraries that target it.
+
+## The past
```

Comment:
I think this section can be removed.  It's only useful for those readers that already know what the past was. They already know, and don't need the information.